src/app/personside/dialogpanel/Hurtigreferat/HurtigreferatContainer.tsx

```diff
@@ -89,9 +84,8 @@ function HurtigreferatContainer(props: Props) {
         tema && setTemaCookie(tema.kodeRef);
```

Comment:
cookie?

Reply:
ja, hva tenker du på?

Reply:
Hvorfor bruker vi det? Hva er formålet liksom?

Reply:
Det er for hurtigreferat, husker hvilket tema du valgte forrige gang for å minimere antall klikk. Ønske fra produkteier og designer. Men tenker at dette bør lagres i saksbehandlerinnstillinger, så kan lage en oppgave på det.